README.adoc

```diff
@@ -33,7 +33,7 @@ So Avro Keyboard users will feel at home with OpenBangla Keyboard in Linux.
 
 OpenBangla Keyboard is the default Bangla writing software for http://www.magpieos.net[MagpieOS], a Bangladeshi Linux distribution.
 
-image:https://img.shields.io/github/workflow/status/bdeshi/openbangla-keyboard/CI/gh-actions[GitHub Workflow Status (branch)] {nbsp} 
+image:https://img.shields.io/github/workflow/status/OpenBangla/OpenBangla-Keyboard/CI[GitHub Workflow Status (branch)] {nbsp}
```

Comment:
I'd be nice if it links to the [CI](https://github.com/OpenBangla/OpenBangla-Keyboard/actions?query=workflow%3ACI) page. Now it links to a image file when clicked.

Reply:
ok, linked to branch specific actions report. also switched image source to github.